Sources/Common/autogenerated/AutoMockable.generated.swift

```diff
@@ -9,6 +9,9 @@ import FoundationNetworking
 #if canImport(UserNotifications)
 import UserNotifications
 #endif
+#if canImport(UIKit)
+import UIKit
+#endif
```

Comment:
Because we modified the `Sources/Templates/AutoMockable.stencil` file, all `AutoMockable.generated.swift` files in the codebase gets this added to it. 

This is required because of `UIKitWrapper` modification.